src/db/results.rs

```diff
@@ -43,7 +43,7 @@ pub struct GetBso {
     pub sortindex: Option<i32>,
     // NOTE: expiry (ttl) is never rendered to clients and only loaded for
     // tests: this and its associated queries/loading could be wrapped in
-    // #[cfg(any(test, feature = "db_test"))]
+    // #[cfg(test)]
```

Comment:
Should this just be removed since it is commented out?

Reply:
It's actually part of the sentence on the previous line

Reply:
Oh right. Lol